src/meadowrun/run_job.py

```diff
@@ -487,6 +487,7 @@ async def mirror_local(
             LocalInterpreter, InterpreterSpecFile, ContainerInterpreterBase, None
         ] = None,
         environment_variables: Optional[Dict[str, str]] = None,
+        additional_extensions: Union[Iterable[str], str, None] = None,
```

Comment:
I don't like this. I rarely want to copy whatever txt or csv files that appear anywhere on sys.path, I typically know exactly what I want to copy. Also your pytest problem may be related to this (wild guess), as pytest does like to manipulate python path.
How about some kind of wildcard syntax, e.g. additional_files = "*/*.csv" for example? You can just resolve it with `glob.glob`.